pkgs/applications/window-managers/phosh/default.nix

```diff
@@ -0,0 +1,130 @@
+{ stdenv
+, pkgs
+, fetchFromGitLab
+, substituteAll
+, meson
+, ninja
+, pkgconfig
+, wrapGAppsHook
+, libhandy
+, pulseaudio
+, glib
+, gtk3
+, gnome3
+, gcr
+, pam
+, upower
+, wayland
+, dbus
+, xvfb_run
+, phoc
+, feedbackd
+, squeekboard
+, networkmanager
+, polkit
+, libsecret
+, git
+, writeText
+, makeDesktopItem
+}:
+
+let
+  gvc = fetchFromGitLab {
```

Comment:
We really need `fetchFromGitLab` to be able to handle submodules like `fetchFromGitHub` is.

Reply:
Is this something we need to do for this PR to get merged...?